polygon-daemon/values.yaml

```diff
@@ -0,0 +1,72 @@
+#values file for the polygon node
+
+replicas: 1
+configMap: scripts-cm
+
+# filesystemId for the storage class
+fileSystemId: ""
+
+chain: mainnet
+
+parts:
```

Comment:
Instead of the parts, maybe keying off name so the values can be easily referenced in the config map scripts?